### ISSUE TITLE
Connect-SFCluster online help (Example 3) for Node connection

### ISSUE THREAD

**Issue by scaleoutsean** [solidfire/PowerShell#65]
```
   -------------------------- EXAMPLE 3 --------------------------
    
    PS C:\>Connect-SFCluster 172.27.1.5 -Node
    
```

Maybe we should move the IP to after `-Node` to make it according to documented parameters, in which FQDN or IP comes after. The example seems confusing to some users.

**Comment by Salmanius** [comment]
Fair point. I'll add a story for us to address this in another example with explanation that both ways are valid.

@scaleoutsean are you suggesting the example be something like:


    PS C:\>Connect-SFCluster -Node 172.27.1.5

**Comment by arjun960** [comment]
@scaleoutsean Help of Connect-SFCluster is updated and the change is available in the Powershell SDK 12.3 release. So closing the issue.
